openlibrary/i18n/__init__.py

```diff
@@ -107,6 +111,8 @@ def update_translations():
             write_po(f, catalog)
             f.close()
             print('updated', po_path)
+        else:
+            print(f"ERROR: {po_path} does not exist...")
 
     compile_translations()
```

Comment:
We should probably remove this entirely, but for now let's just forward it.
```suggestion

    compile_translations(locales_to_update)
```

Reply:
This seems good.  I agree with decoupling updating and compiling in the future.